### ISSUE TITLE
Model output

### ISSUE THREAD

**Issue by NovaMind-Z** [fabiotosi92/monoResMatch-Tensorflow#6]
Hi, 
   Thanks for your amazing work! When I try to test a single image of my own using your pretrained model, the otput disp is a .npy array ranging from 0 to 400. I wonder if the disp means the disparity map or depth map?
Thanks in advance for your reply！

**Comment by wangshui123** [comment]
Hi,
@Zhu-Jinkuan,
hello, I have a problem, I have not solved it. Can you discuss it together? Can you leave your contact information, thank you

**Comment by Zhu-Jinkuan** [comment]
> Hi,
> @Zhu-Jinkuan,
> hello, I have a problem, I have not solved it. Can you discuss it together? Can you leave your contact information, thank you

**Comment by NovaMind-Z** [comment]
> Hi,
> @Zhu-Jinkuan,
> hello, I have a problem, I have not solved it. Can you discuss it together? Can you leave your contact information, thank you
Hi,
Do you have the same problem with me?  I think I have solved my problem. the output of the model is the disparity map. If you want to convert disparity map to depth map, you can use the formula:
depth = baseline*focal/disparity

**Comment by wangshui123** [comment]
Hi,
thank you very much ,can you tell me  how to get the proxy label? Did you get it with the SGM program provided by the author?   
I am a Chinese, do you have WeChat ID, can you add, so that I can consult with you


thank you very much !!




------------------ 原始邮件 ------------------
发件人: "Zhu-Jinkuan"<notifications@github.com>;
发送时间: 2019年10月8日(星期二) 晚上10:11
收件人: "fabiotosi92/monoResMatch-Tensorflow"<monoResMatch-Tensorflow@noreply.github.com>;
抄送: "小冢虎王水"<1925455317@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [fabiotosi92/monoResMatch-Tensorflow] Model output (#6)



 
Hi,
 @Zhu-Jinkuan,
 hello, I have a problem, I have not solved it. Can you discuss it together? Can you leave your contact information, thank you
 Hi,
 Do you have the same problem with me?  I think I have solved my problem. the output of the model is the disparity map. If you want to convert disparity map to depth map, you can use the formula:
 depth = baseline*focal/disparity

**Comment by xuwanpeng12** [comment]
> 
> Hi, thank you very much ,can you tell me how to get the proxy label? Did you get it with the SGM program provided by the author? I am a Chinese, do you have WeChat ID, can you add, so that I can consult with you thank you very much !!

I encountered the same problem as you. Have you solved it? I am also a Chinese student. Can I chat with WeChat? Mine is 343934268. Looking forward to your reply.

**Comment by thgpddl** [comment]
没想到唯一一个issue底下这么多中国人哈哈哈哈
